### DISCUSSION TITLE
Can't import deck.gl in my React TS app

### DISCUSSION THREAD

**Discussion by kilian-goetz** [visgl/deck.gl#7365] [Q&A]
Hello,

I have a react typescript application in which I have installed deck.gl. I followed installation guide and deck.gl is refered inside my package.json. But when I try to import something from deck.gl it mentioned that deck.gl isn't foundable. Someone now why I have got this issue ?

```javascript

import {Layer} from 'deck.gl';

```
 
 Have nice day,
 Kind regards, Kilian GOËTZ.

**Comment by kylebarron** [comment]
I think the preferred import method is from `@deck.gl/core`, `@deck.gl/layers` and so on

**Reply by Pessimistress** [reply]
That should not matter. If `deck.gl` is in your package.json but cannot be found, then there is a problem with your NPM install.

**Reply by kylebarron** [reply]
Good point; sounds like @Beladric needs to redo `npm install` or `yarn install`
